spi/src/main/java/io/apicurio/studio/spi/storage/model/OutboxEvent.java

```diff
@@ -0,0 +1,11 @@
+package io.apicurio.studio.spi.storage.model;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+public interface OutboxEvent {
+    String getId();
+    String getAggregateId();
+    String getAggregateType();
```

Comment:
What are these?  The `aggregateId` seems to be the native ID of the "thing" the event represents?  So in this case the `designId`.  Yes?  What's `aggregateType` ?

Reply:
AggregateId is the id of the design, yes. `aggregateType` is the "type" of the outbox event, it's the value that eventually allows the connector to route the event to the appropriate topic. In the spirit of flexibility of the solution, I think it's worth having it even if we decide to go with the single-topic approach.

Reply:
What are some examples of  the value of `aggregateType`?  And what is the semantic meaning of `aggregate` in these variables?  Not sure I fully understand yet...

Reply:
Aggregate is the source table of the event. In Studio, in this case, we're only tracking the designs, so an example aggregate would be the design. It may be users if we decide to send events for those. Any entity where you're tracking events. The aggregateType is something that defines it, in this case, I'm using "DESIGN", which is usually used to separate the different events into separate topics. But, since we don't want that, we can probably use something like "studio".